Sources/SwiftUIKit/Views/TableView.swift

```diff
@@ -32,6 +32,8 @@ public typealias TableViewCell = DataConfigurable & CellConfigurable & CellUpdat
 
 public typealias TableHeaderFooterViewHandler = (Int) -> UIView?
 public typealias TableHeaderFooterTitleHandler = (Int) -> String?
+public typealias TableDidSelectIndexPathHandler = (IndexPath) -> Void
+public typealias TableHighlightIndexPathHandler = (IndexPath) -> Bool?
```

Comment:
This shouldn't need to be optional. 

`tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool`

Reply:
You are right, I solved it!